src/pudl/metadata/fields.py

```diff
@@ -2241,6 +2249,31 @@
         "type": "boolean",
         "description": "Indicates whether the generator is part of a solid fuel gasification system",
     },
+    "stack_id_eia": {
+        "type": "string",
+        "description": (
+            "The value EIA uses to identify a stack or chimney on a power plant where "
```

Comment:
One non-blocking question: Is this the value used by EIA, or the value that an individual plant owner uses to identify this unit, as reported by the EIA? I wonder if the wording of these descriptions is slightly misleading, as we're intending to point to the data source rather than the original creator of these IDs.

Reply:
This is a great point! These values are reported by the utilities and are thus used by EIA.